frog/frog.scrbl

```diff
@@ -174,15 +174,14 @@ project/
     @comment{# Zero or more other @tt{.md} files, for non-post pages.}
     @comment{# May be in subdirs.}
   css/
-    bootstrap.css            @comment{# \}
-    bootstrap.min.css        @comment{# get these files}
-    bootstrap-theme.css      @comment{# from @url["http://getbootstrap.com"]}
-    bootstrap-theme.min.css  @comment{# /}
+    bootstrap.min.css        @comment{# get these files
```

Comment:
The Travis CI failure is due to the missing closing `}` on this line.

Once you add this, try running `make setup` and `make test` locally to confirm it builds and tests fine. Then when you push the commit, Travis should be all green, and we can merge!

Reply:
Just changed it! My lack of a Scribble Emacs mode may have been the problem and I didn't see this before pushing. Hopefully Travis will be fine (I see about 4 other errors reported under `make test` so hopefully that doesn't stop it)